ogc-example/src/main/resources/application.conf

```diff
@@ -256,19 +256,7 @@ layers = {
         type = "mapalgebrasourceconf"
         name = "addition-house-income"
         title = "test addition"
-        algebra = {
-          "args" : [
-            {
-              "name" : "us-census-median-household-income",
-              "symbol" : "rasterV"
-            },
-            {
-              "name" : "us-census-median-household-income",
-              "symbol" : "rasterV"
-            }
-          ],
-          "symbol" : "+"
-        }
+        algebra = "{\"args\":[{\"name\":\"us-census-median-household-income\",\"symbol\":\"rasterV\"},{\"name\":\"us-census-median-household-income\",\"symbol\":\"rasterV\"}],\"symbol\":\"+\"}"
```

Comment:
Looks really bad. Mb you can write a manual decoder? Should not be too bad.

Reply:
Agreed. I'm working on a JSON reader atm